aerospike/content.md

```diff
@@ -2,174 +2,289 @@
+[Aerospike](http://aerospike.com) is a distributed NoSQL database purposefully designed for high performance applications. Aerospike supports key-value and document data models, and has multiple data types including List, Map, HyperLogLog, GeoJSON, and Blob. Aerospike's patented hybrid memory architecture delivers predictable high performance at scale and high data density per node.
 
 %%LOGO%%
 
 ## Getting Started
 
-Aerospike Enterprise Edition requires a feature key file to start and to ungate certain features in the database, such as compression. Enterprise customers can use their production or development keys.
+Aerospike Database Community Edition (CE) supports the same developer APIs as Aerospike Database Enterprise Edition (EE), except for durable deletes. They differ in ease of operation and [enterprise features](https://aerospike.com/products/features-and-editions/), such as compression.
 
-Anyone can [sign up](https://www.aerospike.com/lp/try-now/) to get an evaluation feature key file for a full-featured, single-node Aerospike Enterprise Edition.
+Since server version 6.1, Aerospike EE starts in a single-node cluster evaluation mode, with all its enterprise features available.
 
-Aerospike Community Edition supports the same developer APIs as Aerospike Enterprise Edition, and differs in ease of operation and enterprise features. See the [product matrix](https://www.aerospike.com/products/product-matrix/) for more.
+### Running an Aerospike Server node
 
-### Running an Aerospike EE node with a feature key file in a mapped directory
+#### Enterprise Edition
 
-```console
+Running Enterprise Edition with default evaluation feature key (versions 6.1+).
+
+```sh
+docker run -d --name aerospike -p 3000-3002:3000-3002 %%IMAGE%%:ee-[version]
+```
+
+Running Enterprise Edition with a feature key file in a mapped directory:
+
+```sh
+docker run -d -v DIR:/opt/aerospike/etc/ -e "FEATURE_KEY_FILE=/opt/aerospike/etc/features.conf" --name aerospike -p 3000-3002:3000-3002 %%IMAGE%%:ee-[version]
+```
+
+Running Enterprise Edition with a feature key file in an environment variable:
+
+```sh
 docker run -d -v DIR:/opt/aerospike/etc/ -e "FEATURE_KEY_FILE=/opt/aerospike/etc/features.conf" --name aerospike -p 3000-3002:3000-3002 %%IMAGE%%:ee-[version]
 ```
 
 Above, *DIR* is a directory on your machine where you drop your feature key file. Make sure Docker Desktop has file sharing permission to bind mount it into Docker containers.
 
-### Running an Aerospike EE node with a feature key file in a mapped directory
+#### Community Edition
 
-```console
-FEATKEY=$(base64 ~/Desktop/evaluation-features.conf)
-docker run -d -e "FEATURES=$FEATKEY" -e "FEATURE_KEY_FILE=env-b64:FEATURES" --name aerospike -p 3000-3002:3000-3002 %%IMAGE%%:ee-[version]
-```
+By using Aerospike Community Edition you agree to the [COMMUNITY_LICENSE](community/COMMUNITY_LICENSE).
```

Comment:
Just one minor bit of feedback. This link isn't going to work here or on Docker Hub. Can you update it to a full URL?